### DISCUSSION TITLE
训练自己的数据集报错

### DISCUSSION THREAD

**Discussion by mstd-jie** [open-mmlab/mmocr#1686] [General]
出现shapely.errors.GEOSException: TopologyException: Input geom 0 is invalid: Self-intersection at 434.23054389410925 232.05115873816914这种错误，但我看标注是没问题的呀

**Comment by gaotongxiao** [comment]
There are some invalid polygons in your annotations. You can add `FixInvalidPolygons` to both your train and test pipeline to fix this issue.

Here is an example:
https://github.com/open-mmlab/mmocr/blob/1.x/configs/textdet/dbnet/dbnet_resnet18_fpnc_1200e_totaltext.py
